lib/app/custom_widgets/match_league_and_timer_badge.dart

```diff
@@ -1,24 +1,23 @@
 import 'package:cached_network_image/cached_network_image.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_svg/flutter_svg.dart';
+import 'package:score_live/core/applocalization_context.dart';
 import 'package:score_live/models/live_match_model/live_match_model.dart';
 import 'package:score_live/presentation/constants/app_colors.dart';
+import 'package:score_live/presentation/constants/text_styles.dart';
 
-class MatchLeagueInfoAndTimerWidget extends StatelessWidget {
-  const MatchLeagueInfoAndTimerWidget({
-    super.key,
-    required this.leagueName,
-    required this.liveMatch,
-  });
+class MatchLeagueAndTimerBadge extends StatelessWidget {
+  const MatchLeagueAndTimerBadge({super.key, required this.liveMatch});
 
-  final String leagueName;
   final LiveMatchModel liveMatch;
 
   @override
   Widget build(BuildContext context) {
+    const String stringPlaceholder = '';
```

Comment:
Do you really need this string placeholder variable? As far as I can see it is used in just one place.

Reply:
its used in many places, will move it to separate file with variables similar to this one